### PR TITLE
Update license to indicate GPL dependency.

### PR DESCRIPTION
**Pull request by rcurtin**
This comes from #441.  Happy to update the language---primarily I just wanted to indicate that there is indeed a GPL dependency here for now. :+1:

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -22,3 +22,8 @@ The GLM module is licensed under the MIT License:
 > LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 > OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 > WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Note that the GLM module has an implicit dependency on the
```
Comment (nalimilan) [inline LICENSE.md:26]: Not sure what "implicit" means here:
```suggestion
Note that the GLM module depends on the
```

Comment (rcurtin) [reply LICENSE.md:26]: Hm, yeah, maybe there is a better word.  GLM.jl doesn't depend *directly* on Rmath, the dependency is via Distributions via StatsFuns.  Anyway, if you prefer I can take your suggestion, just let me know what you think now that I've clarified what I was going for. :)

Comment (nalimilan) [reply LICENSE.md:26]: Let's say indirect then. :-)
```suggestion
Note that the GLM module has an indirect dependency on the
```

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**